### PR TITLE
PROD-22 Modal and book to stotybook

### PR DESCRIPTION
**Pull request by Valdas-Stonkus**
Added modal and Book components to storybook

### CODE REVIEW

File: src/Components/Button/Button.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    title: 'Button component',
+    component: Button
+}
+
+const Template = (args) => {
+    const {
+        className, onClick, children, ariaLabel
+    } = args
+    return (
+        <div>
```
Comment (prodevshub) [inline src/Components/Button/Button.stories.jsx]: In this case it doesn't matter but usually try not to leave unnecessary html tag.

File: src/Components/Button/Button.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <Button className={className} onClick={onClick} ariaLabel={ariaLabel}>{children}</Button>
```
Comment (prodevshub) [inline src/Components/Button/Button.stories.jsx]: If prop and variable name is the same, usually we are using this syntax: 
`            <Button
                {...{
                    className,
                    onClick,
                    ariaLabel
                }}
            >
                {children}
            </Button>`

File: src/Components/Modal/Modal.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    return (
+        <div className="App">
+            <h1>Modal demo</h1>
+            <button onClick={() => setShow(true)}>Show Modal</button>
+
+            <Modal
+                show={show}
+                title={title}
+                onSubmit={onSubmit}
+                onClose={() => setShow(false)}
```
Comment (prodevshub) [inline src/Components/Modal/Modal.stories.jsx:25]: If you have onSubmit function which does the same onClose and onSubmit, jus reuse that function here too.

File: src/Components/Modal/Modal.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    const { title, children } = args
+
+    return (
+        <div className="App">
+            <h1>Modal demo</h1>
+            <button onClick={() => setShow(true)}>Show Modal</button>
+
+            <Modal
+                show={show}
```
Comment (prodevshub) [inline src/Components/Modal/Modal.stories.jsx]: Same comment with props as above:
`
            <Modal
                {...{ show, title, onSubmit }}
                onClose={() => setShow(false)}
            >
`

**Review by prodevshub — APPROVED** [review]

File: src/Components/Button/Button.stories.jsx (hunk 1 above)
Comment (Valdas-Stonkus) [reply src/Components/Button/Button.stories.jsx]: My Eslin auto formater converting like this, and I not save in other format. Try to change it?

File: src/Components/Button/Button.stories.jsx (hunk 2 above)
Comment (Valdas-Stonkus) [reply src/Components/Button/Button.stories.jsx]: Thanks, done.

File: src/Components/Modal/Modal.stories.jsx (hunk 4 above)
Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.stories.jsx]: Done

Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.stories.jsx]: Done

File: src/Components/Button/Button.stories.jsx (hunk 1 above)
Comment (Valdas-Stonkus) [reply src/Components/Button/Button.stories.jsx]: Ohm I understood. You wrote regard <div> tag. Done

File: src/Components/Modal/Modal.stories.jsx (hunk 3 above)
Comment (Valdas-Stonkus) [reply src/Components/Modal/Modal.stories.jsx:25]: Done

**Review by prodevshub — APPROVED** [review]

**Pull request merged**